features/required_tags_starts_with.feature

```diff
@@ -0,0 +1,154 @@
+Feature: Required Tags Starts With
+  As a tester I dont want to miss certain tags on my scenarios
+
+  Background: Prepare Testee
+    Given a file named ".gherkin_lint.yml" with:
+    """
+    ---
```

Comment:
Configuration is tested in rspec. This is setup for the integration test for the new linter and this was the only way we saw to support your 'disable all' under the new structure without significant refactoring.  I'm open to other suggestions.